src/DataGrid.tsx

```diff
@@ -504,6 +490,11 @@ function DataGrid<R, SR, K extends Key>(
   }
 
   function handleKeyDown(event: React.KeyboardEvent<HTMLDivElement>) {
+    if (!(event.target instanceof Element)) return;
+    const isCellEvent = event.target.closest('[role="gridcell"]') !== null;
```

Comment:
Not very react way but works in our case

Reply:
Any meaningful difference between the two?
```suggestion
    const isCellEvent = event.target.closest('.rdg-cell') !== null;
```